docs/configuration.md

```diff
@@ -0,0 +1,63 @@
+# Configuring Voici
+
+## Templates
+
+You can different templates for your Voici application.
+
+Templates can modify the layout and the appearance of your Voici application. Here are a few template examples:
+
+- [Voila Material](https://github.com/voila-dashboards/voila-material): Material design template for Voilà
+- [Voila Vuetify](https://github.com/voila-dashboards/voila-vuetify): Dashboard template for Voilà based on VuetifyJS
```

Comment:
Out of curiosity, have you tested that voila-vuetify works with voici?

Reply:
Partially:

![image](https://user-images.githubusercontent.com/591645/231750541-54144ce3-a627-48ac-a585-820031dd224d.png)


Do you think we should remove it from the docs for now? Do we have another template that already works with Voici?

Reply:
Thanks!

Yes, let's remove it from the docs for now and figure out if we can fix its support separately. Can you open an issue for this?

The reveal and gridstack templates do work AFAIK

Reply:
Yes gridstack looks good:

![image](https://user-images.githubusercontent.com/591645/231752065-13d7fc02-7908-4a71-9c10-845fb07324b5.png)

I'll update the docs to mention this one instead.

Reply:
> Can you open an issue for this?

Probably the main issue is the compatibility with Voila 0.5.0. In this case there should already be one tracking this: https://github.com/voila-dashboards/voila-vuetify/issues/64